monero/.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
```

Comment:
Is this file picked-up by Github CI? I think you will need to move it one folder up.

Reply:
oh that shouldn't be in there at all.  Thanks